ui/src/app/pages/data/data-set/export-data-set-modal.tsx

```diff
@@ -129,7 +129,7 @@ class ExportDataSetModal extends React.Component<
     // Open notebook in a new tab and close the modal
     const notebookUrl = '/workspaces/' + workspaceNamespace + '/' + workspaceFirecloudName +
         '/notebooks/' + appendNotebookFileSuffix(encodeURIComponent(this.state.notebookName));
-    window.open(notebookUrl);
+    window.open(notebookUrl, '_blank');
```

Comment:
Would it make sense to pull this and possibly related functions into a utility class for window/navigation management? Also, defining that string constant someplace would help.

Reply:
Since this is a JS/web standard thing, as opposed to something in our code, I don't know if I would pull it out into a utility class/bother wrapping it in our own method. I don't have a strong feeling though, could go either way.

Reply:
Yes, it is a standard built into the Javascript browser API. Wrapping it will only result in misdirection/obfuscation, and would not be portable to any other code project.